xla/service/gpu/ir_emitter_nested.cc

```diff
@@ -306,12 +307,23 @@ llvm::Value* AddrCastToDefault(llvm::Value* arg, llvm::IRBuilder<>& b) {
   return arg;
 }
 
-void EmitAMDGPUAtomicAdd(llvm::IRBuilder<>* builder,
-                         llvm::Value* output_address, llvm::Value* source) {
+void EmitAMDGPUAtomicFAdd(llvm::IRBuilder<>* builder,
+                          IrEmitterContext& ir_emitter_context,
+                          llvm::Value* output_address, llvm::Value* source,
+                          PrimitiveType element_type) {
+  static bool no_paired_atomics = []() -> bool {
+    const char* flag = std::getenv("ROCM_NO_PAIRED_ATOMICS");
```

Comment:
I'm guessing this should be in `XLA_FLAGS`. If for some reason it can't, let's at least shorten this to `return flag && ...`.